docs/releasing/testing-and-linting.md

```diff
@@ -141,12 +141,24 @@ This will update the snapshot file. Commit this file separately with a commit me
 
 ## Visual regression testing with [Percy](https://percy.io/)
 
-We generate 2 screenshots for each default example of every component. One example has JavaScript enabled, the other has JavaScript disabled. Screenshots are not taken for all the different variations of each component. This tool is not a replacement for manual testing.
+We generate at least one screenshot per component. Typically this will be the default example but also screenshot additional examples for distinct states of components eg: small radios or checkboxes.
+
+We tell Percy we want to screenshot an example by adding the `screenshot` attribute to an example in a component's yaml fixture. If `screenshot` is either true or has a `variants` attribute containing an array of valid options ('default' or 'No-JS') then Percy will screenshot that example.
```

Comment:
One sentence explanation of `variants`? ie: that they modify the screenshot in some way (for example, default just takes the same screenshot as though you'd set `screenshot: true`, whereas `no-js` will disable JavaScript before taking the sceenshot)